src/client/src/mockData/mockVsCodeApi.ts

```diff
@@ -84,7 +94,46 @@ const mockVsCodeApi = () => ({
                 locations: [{ label: WEST_US, value: WEST_US }],
                 resourceGroups: [
                   { label: RESOURCE_GROUP_MOCK, value: RESOURCE_GROUP_MOCK },
-                  { label: "ResourceGroupMock2", value: "ResourceGroupMock2" }
+                  {
+                    label: RESOURCE_GROUP_MOCK_2,
+                    value: RESOURCE_GROUP_MOCK_2
+                  },
+                  {
+                    label: RESOURCE_GROUP_MOCK_3,
+                    value: RESOURCE_GROUP_MOCK_3
+                  },
+                  {
+                    label: RESOURCE_GROUP_MOCK_4,
+                    value: RESOURCE_GROUP_MOCK_4
+                  },
+                  {
+                    label: RESOURCE_GROUP_MOCK_5,
+                    value: RESOURCE_GROUP_MOCK_5
+                  },
+                  {
+                    label: RESOURCE_GROUP_MOCK_6,
+                    value: RESOURCE_GROUP_MOCK_6
+                  },
+                  {
+                    label: RESOURCE_GROUP_MOCK_7,
+                    value: RESOURCE_GROUP_MOCK_7
+                  },
+                  {
+                    label: RESOURCE_GROUP_MOCK_8,
+                    value: RESOURCE_GROUP_MOCK_8
+                  },
+                  {
+                    label: RESOURCE_GROUP_MOCK_9,
+                    value: RESOURCE_GROUP_MOCK_9
+                  },
+                  {
+                    label: RESOURCE_GROUP_MOCK_10,
+                    value: RESOURCE_GROUP_MOCK_10
+                  },
+                  {
+                    label: RESOURCE_GROUP_MOCK_11,
+                    value: RESOURCE_GROUP_MOCK_11
+                  }
```

Comment:
What do all these mocks mean?

Reply:
I added this to make test the longer dropdowns since we set a limit on how long the dropdown is in the css

Reply:
Why not just use a loop to generate this?

Reply:
I could, but I don't think it's necessary. Do you guys have any thoughts on this? @njindal239 @jasmingoh

Reply:
Agree with @jsondoo, I don't think it is necessary to create a loop and it doesn't really make sense. Maybe we can just use the same constant for all labels and values since this is just a mock?

Reply:
Yeah using the same value works, its just a lot of unnecessary noise I feel. Any way you can reduce it is good. The map/loop is just the first thing that came to mind due to the repeated elements with only a number being different.

Reply:
@SahilTara Yea, I like this solution the most